torch/csrc/jit/codegen/cuda/test/test_gpu3.cpp

```diff
@@ -6588,6 +6588,152 @@ TEST_F(NVFuserTest, FusionVectorizeRepro1843_CUDA) {
   testValidate(fusion, cg_outputs, {t1, t0}, {ref}, __LINE__, __FILE__);
 }
 
+// Repro for
+// https://github.com/csarofeen/pytorch/issues/2094
+TEST_F(NVFuserTest, FusionRepro2094_CUDA) {
+  return;
```

Comment:
This repro is generated by `cpp-repro-gen.py`, I put it into this PR for convenience, but it is skipped as it is failing.